lib/core/base-provider.js

```diff
@@ -26,10 +26,20 @@ class Provider {
 
     let providersToLoad = {};
 
-    if (options !== undefined && options.hasOwnProperty("overrideProviders")) {
+    if (
+      options !== null &&
+      options !== undefined &&
+      options.hasOwnProperty("overrideProviders")
+    ) {
       ncConfig.map(provider => {
         const module = provider.plugin;
-        providersToLoad[provider.tag] = module(provider.configPath);
+        try {
+          providersToLoad[provider.tag] = module(provider.configPath);
+        } catch (err) {
+          throw new Error(
+            "It seems you have not added any valid cloud provider"
```

Comment:
Let's rename the error as '🤖 It seems you have not added a valid cloud provider'

Reply:
@rajikaimal Can you tell how to add robot emoji in error message? I am trying with ```throw new Error(`${robot} It seems you have not added a valid cloud provider`)``` but it's not working. Can I add error message without emoji?

Reply:
Well, you just have to copy and paste that emoji. That should work.

Reply:
@Cheesetouched I tried that but it not working in Error messages

Reply:
@rajikaimal , @Cheesetouched done